src/api/gameTitles/gameTitles.repository.js

```diff
@@ -1,22 +1,35 @@
-import gameTitleModel from './gameTitles.model.js';
```

Comment:
I understand foregoing CamelCase with the genres_gameTitles router, but I'm not sure why you've done it here

Reply:
Eslint warns that "constructor" functions should be named in UpperCamelCase.  The models are constructor functions, but eslint warning is triggered only when you use the 'new' keyword to invoke the function...
Sadly we have several constructor functions in camelCase instead of UpperCamelCase, I've corrected some of them but not all.